inst/include/interface/rcpp/rcpp_objects/rcpp_fleet.hpp

```diff
@@ -125,46 +132,82 @@ class FleetInterface : public FleetInterfaceBase {
    */
   ParameterVector log_Fmort;
   /**
-   * @brief The vector of natural log of the expected index of abundance for the fleet.
+   * @brief The vector of natural log of the expected total landings for 
+   * the fleet.
    */
-  ParameterVector log_expected_index;
+  ParameterVector log_landings_expected;
   /**
-   * @brief The vector of expected catch-at-age in numbers for the fleet.
+   * @brief The vector of natural log of the expected index of abundance 
+   * for the fleet.
    */
-  ParameterVector proportion_catch_numbers_at_age;
+  ParameterVector log_index_expected;
   /**
-   * @brief The vector of expected catch-at-length in numbers for the fleet.
+   * @brief The vector of expected landings-at-age in numbers for the fleet.
    */
-  ParameterVector proportion_catch_numbers_at_length;
+  ParameterVector composition_numbers_at_age;
   /**
-   * @brief The vector of conversions to go from age to length, i.e., the age-to-length-conversion matrix.
+   * @brief The vector of expected landings-at-length in numbers for the fleet.
    */
-  ParameterVector age_length_conversion_matrix;
+  ParameterVector composition_numbers_at_length;
   /**
-   * @brief Should catchability (q) be estimated? The default is false.
+   * @brief The vector of expected landings-at-age in numbers for the fleet.
+   */
+  ParameterVector proportion_numbers_at_age;
+  /**
+   * @brief The vector of expected landings-at-length in numbers for the fleet.
+   */
+  ParameterVector proportion_numbers_at_length;
+  /**
+   * @brief The vector of conversions to go from age to length, i.e., the 
+   * age-to-length-conversion matrix.
+   */
+  ParameterVector age_to_length_conversion;
+  /**
+   * @brief Should index scaling parameter (q) be estimated? 
+   * The default is false.
    */
   SharedBoolean estimate_q = false;
   /**
-   * @brief Is catchability (q) a random effect? The default is false.
+   * @brief Is landingsability (q) a random effect? The default is false.
```

Comment:
this nomenclature seems odd.

Reply:
Hahahaha I was confused by landingsability too! didn't realize it was my own replace function that caused it!! We can go back to catchability.

Reply:
I often "over-replace" too. I noticed it happens elsewhere in the code so probably need to do a global search for landingsability.

Reply:
also "landingses"